terraform/monitoring/panels/app/http_request_latency.libsonnet

```diff
@@ -15,6 +15,25 @@ local targets   = grafana.targets;
         .withUnit('ms')
     )
 
+    .setAlert(vars.environment, grafana.alert.new(
+      namespace     = vars.namespace,
+      name          = '%(env)s - HTTP request latency too high' % { env: vars.environment },
+      message       = '%(env)s - HTTP request latency too high' % { env: vars.environment },
+      notifications = vars.notifications,
+      noDataState   = 'no_data',
+      conditions    = [
+        grafana.alertCondition.new(
+          evaluatorParams = [ 5000 ],
```

Comment:
Looks like the 5000 is too high for an alert, probably we should start alerting at 2000-3000

Reply:
Changed to 3000 in https://github.com/WalletConnect/notify-server/pull/297#discussion_r1458894962